contrib/vagrant/start.sh

```diff
@@ -49,12 +53,22 @@ EOF
         fi
         hexIPv4=$(printf "%02X" "${i}")
         hexIPv6=$(printf "%04X" "${index}")
-        cat <<EOF >> "${filename}"
+
+	if [ "$LB" = 1 ] && [ $((node_index%2)) -eq 0 ]; then
```

Comment:
@aanm: It means if we are in load balancer mode $LB=1 and an even node %2 we are a client node and we set the route index to point to the load balancer node. For odd numbered nodes the route index will point to the actual node just like when LB=0, i.e the behavior today.